### PR TITLE
Project rename

### PR DESCRIPTION
**Pull request by MarconZet**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ jobs:
       # https://circleci.com/docs/2.0/configuration-reference/#available-machine-images
       image: ubuntu-2004:202101-01
 
-    working_directory: ~/akka-safer-serializer
```
Comment (PawelLipski) [inline .circleci/config.yml:8]: Looks like the only file with occurrences of `git grep -i safer` left is README.md

Comment (MarconZet) [reply .circleci/config.yml:8]: Yes, I already rebased #46

File: borer-akka-serializer/src/main/scala-2.12/org/virtuslab/akkaserializationhelper/RuntimeReflections.scala [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package org.virtuslab.akkasaferserializer
+package org.virtuslab.akkaserializationhelper
```
Comment (PawelLipski) [inline borer-akka-serializer/src/main/scala-2.12/org/virtuslab/akkaserializationhelper/RuntimeReflections.scala:1]: Just thinking whether this should be called `org.virtuslab.ash` in package names 🤔

Comment (MarconZet) [reply borer-akka-serializer/src/main/scala-2.12/org/virtuslab/akkaserializationhelper/RuntimeReflections.scala:1]: I agree

File: borer-akka-serializer/src/main/scala-2.12/org/virtuslab/ash/RuntimeReflections.scala [hunk 3]
```diff
@@ -1,4 +1,4 @@
-package org.virtuslab.akkasaferserializer
+package org.virtuslab.ash
```
Comment (PawelLipski) [inline borer-akka-serializer/src/main/scala-2.12/org/virtuslab/ash/RuntimeReflections.scala:1]: I checked, it seems there aren't prominent projects called `ash` in Scala open source... this is the closest I could find: https://github.com/shadaj/ash but it's completely abandoned

**Review by PawelLipski — APPROVED** [review]
Approving, subject to the rename of the entire repo

**Pull request merged**